packages/cli/src/commands/nuke.ts

```diff
@@ -56,7 +56,7 @@ export default class Nuke extends Command {
     const { flags } = this.parse(Nuke)
     if (initializeEnvironment(logger, flags.environment)) {
       await runTasks(
-        askToConfirmRemoval(new Prompter(), flags.force, compileProjectAndLoadConfig()),
+        askToConfirmRemoval(new Prompter(), flags.force, compileProjectAndLoadConfig(process.cwd())),
```

Comment:
Do we really need to call `process.cwd()` here and pass the result as a value to `compileProjectAndLoadConfig`? Calling to it from `compileProjectAndLoadConfig` should make no difference. AFAIK, the process doesn't change over the whole program unless you call to `chdir`

Reply:
I think I get your point. Yes, we need to pass now the directory we want to compile, as `compileProjectAndLoadConfig` is called in other places in the code (when deploying, for example) with a different path than CWD (the path of the blackbox project). I did this so that we don't need to use `process.chdir` 😃